regression/barebones/barebones_test.go

```diff
@@ -19,38 +16,23 @@ var _ = Describe("Barebones Test", func() {
 	var (
 		fabricTestDir        string
 		testDataDir          string
-		pteReportCmd         string
-		pteReportTxt         string
-		pteFinalReport       string
-		expectedInvokeStatus string
-		expectedQueryStatus  string
 
 		action              string
 		networkSpecPath     string
 		inputSpecPath       string
-		inputSpecPathMaster string
 
 		kubeConfig    string
 		envKubeConfig bool
 		containerType string
 	)
 
-	BeforeEach(func() {
+	BeforeSuite(func() {
```

Comment:
for best practice and to make all test suites similar, BeforeSuite and AfterSuite can be moved barebones_suite_test.go.

Reply:
will change it to be consistent with other tests